cmd/webpkgserver/webpkgserver.example.toml

```diff
@@ -202,6 +202,84 @@
   # bytes for the OCSP response.
   #AllowTestCert = false
 
+# IMPORTANT NOTE: the support of the ACME protocol and automatic renewal of
+# certificates is currently in the EXPERIMENTAL stage.  Once we have more
+# experience with people using it out in the wild, we will gradually move it to
+# the PRODUCTION stage.
+#
+# ACME is a protocol that allows automatic renewal of certificates.
```

Comment:
The paragraph should start with a sentence to indicate "this section is meant for ACME config," then explain ACME protocol. Otherwise the reader might get confused as they see the ACME protocol explained _suddenly_.

Reply:
Added.

Reply:
Done.